apps/meteor/app/statistics/server/lib/statistics.ts

```diff
@@ -406,6 +406,8 @@ export const statistics = {
 				}),
 		);
 
+		statistics.fileStoreType = settings.get('FileUpload_Storage_Type');
```

Comment:
Post-merge review: would have been better to add this to the `settings` key of stats, where the rest of settings are.